dashboard/config/upstream-kasan.config

```diff
@@ -5803,7 +5810,6 @@ CONFIG_COUNTER=y
 #
 CONFIG_DCACHE_WORD_ACCESS=y
 CONFIG_VALIDATE_FS_PARSER=y
-CONFIG_FSINFO=y
```

Comment:
The config needs to be generated on linux-next, or we will lose this coverage on linux-next.

Reply:
Oh sorry I thought I checked-out linux-next that is recent enough.